### PR TITLE
Add developerMode method

### PR DESCRIPTION
**Pull request by michaelcfine**
Added the developerMode property method

### CODE REVIEW

File: src/modules/device.json [hunk 1]
```diff
@@ -97,6 +97,37 @@
...
                 }
             ]
+        },
+        {
+            "name": "developerMode",
+            "tags": [
+                {
+                    "name": "property",
+                    "x-manages": [
+                        "xrn:firebolt:capability:settings:device"
```
Comment (jlacivita) [inline src/modules/device.json:107]: We want a specific capability for developer mode, imo.

Propose: `xrn:firebolt:capability:security:developer-mode`

**Review by jlacivita — CHANGES_REQUESTED** [review]

File: src/modules/device.json [hunk 2]
```diff
@@ -97,6 +97,37 @@
                     }
                 }
             ]
+        },
+        {
+            "name": "developerMode",
+            "tags": [
+                {
+                    "name": "property",
```
[the remark this answers is not in the document]
Comment (michaelcfine) [reply src/modules/device.json]: Had copied that code; there's lots of it elsewhere that should be fixed. Have separated into two separate tags.

**Comment by jlacivita** [comment]
Adding Developer Mode setting to Device module.

**Pull request merged**